_posts/en/newsletters/2019-03-26-newsletter.md

```diff
@@ -0,0 +1,256 @@
+---
+title: 'Bitcoin Optech Newsletter #39'
+permalink: /en/newsletters/2019/03/26/
+name: 2019-03-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a proposal to encrypt P2P communication
+and describes Lightning Loop, a tool and service for withdrawing
+bitcoins from an LN channel to an onchain transaction.  Also included
+are links to resources about bech32 adoption, summaries of popular
+questions and answers from Bitcoin StackExchange, and a list of notable
+code changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC2:** The second Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  Testing is still needed by organizations and experienced users who
+  plan to run the new version of Bitcoin Core in production.  Use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
+
+## News
+
+- **Version 2 P2P transport proposal:** Jonas Schnelli sent a proposed
+  BIP to the Bitcoin-Dev mailing list that specifies an algorithm to be
+  used to encrypt traffic between peers.  It also specifies some other
+  minor changes to the creation of protocol messages, such as allowing
+  peers to use bandwidth-saving short identifiers and eliminating the
+  SHA256-based checksum on messages, as the [AEAD][]-based encryption
+  scheme protects data integrity.  The proposal is meant to replace
+  [BIP151][] and it contains links to an example implementation for
```

Comment:
There's a for loop in references.md that creates links for all BIP numbers 1 to 400 (covering the entire currently-assigned range) so that I don't have to manually create a ref for each new BIP number Luke assigns.  Yeah, I'm that lazy.  :-)